tinker/static/assets/js/tinker.js

```diff
@@ -14,4 +14,117 @@ $(document).ready(function () {
             $(this).text("- Less Search Filters")
         }
     });
-});
\ No newline at end of file
+});
+
+function pagination() {
+    var numberOfItems = $(" #loop .list-group").length;
+    var limitPerPage = $("#first").val();
+
+    $("#selected-option").change(function() {
+        var o = $(this).find("option:selected");
+        if (o.val() === "All") {
+            limitPerPage = numberOfItems;
+        } else {
+            limitPerPage = o.val()
+        }
+
+        $(".pagination").children("li.to-remove").remove();
+
+        var totalPages = Math.ceil(numberOfItems / limitPerPage);
```

Comment:
is this going to error if there are none to view? since divide by 0.

Reply:
Added in a if statement that checks first if the numberOfItems > 0, if it isn't, no pagination is done.